evap/student/templates/student_vote.html

```diff
@@ -101,11 +104,29 @@
             // hide questionnaire for contributor
             $("#" + contributorId).collapse("hide");
             $("a[href=#" + contributorId + "]").addClass("collapsed");
+
+            // setting the 'checked' prop does not trigger a change event, so manually save all data here
+            sisyphus.saveAllData()
         }
         function selectedAnswer(fieldName) {
             // remove error highlighting when an answer was selected
             $(".choice-error[name=" + fieldName + "]").removeClass("choice-error");
         }
         preventAccidentalClosing = true;
+
+        $(function() {
+            sisyphus = $("#student-vote-form").sisyphus({
```

Comment:
can you replace the variable definition in line 93 above with `$("#student-vote-form").sisyphus({...});` and remove the rest?